src/fsharp/ErrorLogger.fs

```diff
@@ -273,7 +273,7 @@ module ErrorLoggerExtensions =
     open System.Reflection
 
     // Instruct the exception not to reset itself when thrown again.
-    // Why don?t we just not catch these in the first place? Because we made the design choice to ask the user to send mail to fsbugs@microsoft.com. 
+    // Why don't we just not catch these in the first place? Because we made the design choice to ask the user to send mail to fsbugs@microsoft.com. 
```

Comment:
Design Note:      This enables the compiler to prompt the user to send mail to fsbugs@microsoft.com.